routes/api-v1.0/club-route.js

```diff
@@ -0,0 +1,74 @@
+/**
+ * Created by Baby_Moico on 25/05/2015.
+ */
+var express = require('express');
+var route = require('./routes');
+var ClubModel = require('../../models/club-model');
+var Q = require('q');
+var __bind =function(fn, me){ return function(){ return fn.apply(me, arguments); }; };
```

Comment:
Each line must have 80 characters as maximum. Please read this document:
https://google-styleguide.googlecode.com/svn/trunk/javascriptguide.xml